compiler/rustc_feature/src/removed.rs

```diff
@@ -180,6 +180,9 @@ declare_features! (
     /// Allows using items which are missing stability attributes
     (removed, unmarked_api, "1.0.0", None, None, None),
     (removed, unsafe_no_drop_flag, "1.0.0", None, None, None),
+    /// Allows `union` fields that don't implement `Copy` as long as they don't have any drop glue.
+    (removed, untagged_unions, "1.13.0", Some(55149), None,
```

Comment:
No idea if this is the version where it got added, or removed.

It would be really useful if these feature tables had column headers...

Reply:
My guess is that it's a mix -- some of these look like they're using added version number, some the removed version number.